CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Changelog
 
-## Unreleased
+<!-- ## Unreleased -->
+
+## [23.1.0] - 2018-08-02
 
 ### Fixed
-* Updated `typescript-eslint-parser` dependency to version 17.0.1 in order to support TypeScript 3.
-* `plugin:shopify/prettier` and `plugin:shopify/typescript-prettier` now defer Prettier's config to project's `.prettierrc`.
+* Updated `typescript-eslint-parser` dependency to version 17.0.1 in order to support TypeScript 3. ([#121](https://github.com/Shopify/eslint-plugin-shopify/pull/121))
+* Removed default prettier configurations. `plugin:shopify/prettier` and `plugin:shopify/typescript-prettier` now defer Prettier's config to project's `.prettierrc`. ([#121](https://github.com/Shopify/eslint-plugin-shopify/pull/121))
```

Comment:
The apostrophe is indicating a singular project, but the sentence reads like it's talking about many projects.

Use either `a project's .prettierrc` or switch up to a address all projects with `projects' .prettierrcs`.